models/sipnet/tests/testthat/test-write.events.SIPNET.R

```diff
@@ -0,0 +1,36 @@
+context("write.events.SIPNET")
+
+# Helper to remove excess whitespace
+norm <- function(x) gsub("\\s+", " ", trimws(x))
+
+testthat::test_that("write.events.SIPNET produces expected lines", {
+    ev_json1 <- testthat::test_path("event_fixtures/events_site1.json")
+    outdir <- withr::local_tempdir()
+    files <- write.events.SIPNET(ev_json1, outdir)
+    expect_length(files, 1)
+    got <- readLines(files[1])
+    expected <- c(
+        "2022  35  till  0.2",
+        "2022  40  till  0.1",
+        "2022  40  irrig  5 1",
+        "2022  40  fert   0 0 10",
+        "2022  50  plant  10 3 2 5",
+        "2022  250 harv   0.1"
+    )
+    expect_equal(norm(got), norm(expected))
```

Comment:
What's generating the whitespace differences that norm() is addressing here? Seems like this case should be deterministic enough to chase them down and eliminate them